common.mk

```diff
@@ -56,6 +58,7 @@ include $(base_dir)/generators/cva6/cva6.mk
 include $(base_dir)/generators/tracegen/tracegen.mk
 include $(base_dir)/generators/nvdla/nvdla.mk
 include $(base_dir)/tools/dromajo/dromajo.mk
+include $(base_dir)/tools/torture/torture.mk
 
```

Comment:
Instead of placing the make fragment in the torture repo, can you place it under tools? `/tools` will have `/tools/torture/` and `/tools/torture.mk`. I want to avoid polluting the torture repo with chipyard-specific stuff, since the torture tool is generic.